packages/offix-conflicts-server/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "roots": [
+      "<rootDir>/test"
+    ],
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
```

Comment:
Thank you for cleaning up all the commented out code. Not sure if that line is needed but lets keep it